lib/themes/makeBraidTheme.ts

```diff
@@ -237,9 +237,9 @@ const makeWebFonts = (tokens: TreatTheme) => {
   }
 
   const weights = values(tokens.typography.fontWeight);
-  const linkTag = `<link href="https://fonts.googleapis.com/css?family=${name}:${weights
-    .sort()
-    .join(',')}" rel="stylesheet" />`;
+  const linkTag = `<link href="https://fonts.googleapis.com/css?family=${encodeURIComponent(
```

Comment:
🤔 Was this causing an issue?

Reply:
Validator picked it up the colon in there, seems to work before and after in chrome, but figured it was worth fixing.